js/main.js

```diff
@@ -1,5 +1,7 @@
 /* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
```

Comment:
You should remove these eslint directives. If you rely on a variable from another file, you can use this:
```js
/* global restaurants */
```
But if you turn off `no-undef` and `no-unused-vars`, you might do something you didn't intend and the linter can't help you 😢